src/main/java/com/github/mproberts/rxtools/map/SubjectMap.java

```diff
@@ -314,9 +314,9 @@ public void setFaultHandler(Function<K, Single<V>> faultHandler)
     }
 
     /**
-     * Runs the specified Runnable if an observable is associated with the specified key.
+     * Re-emits a fault for the specified key if there is someone bound
      */
-    public void runIfBound(K key, final Runnable runnable)
+    public Completable faultIfBound(K key)
     {
         _readLock.lock();
```

Comment:
shouldn't we wrap this entire method in a Completable.create? you could run into weird side effects when chaining it with flatMap and all

Reply:
Feels like this should just be the exact same as `processNewFaultForKey`